images/nginx/rootfs/build.sh

```diff
@@ -441,6 +442,10 @@ WITH_MODULES="--add-module=$BUILD_PATH/nginx-http-auth-digest-$NGINX_DIGEST_AUTH
   --add-module=$BUILD_PATH/nginx_ajp_module-${NGINX_AJP_VERSION} \
   --add-module=$BUILD_PATH/ngx_brotli"
 
+make
```

Comment:
Why do you need this `make` now?

Reply:
This is temporal. It will be removed after the next release of openresty